### PR TITLE
fix: improve ux for editing snapshot asset

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

In the add asset form, in snapshot editing:
1. in the form, allow any assets to be selected,
2. if the asset already exists in the snapshot, ask the user if they want to edit that entry instead 

image

### CODE REVIEW

File: frontend/app/src/components/dashboard/EditBalancesSnapshotTable.vue [hunk 1]
```diff
@@ -441,12 +481,52 @@ const tableContainer = computed(() => {
...
+            :enable-association="false"
+          />
+          <div v-else>
+            <nft-details
+              :identifier="conflictedBalanceSnapshot.assetIdentifier"
+              :class="css.asset"
+            />
+          </div>
+        </v-sheet>
+      </confirm-dialog>
```
Comment (kelsos) [inline frontend/app/src/components/dashboard/EditBalancesSnapshotTable.vue:529]: Do you think we can split the dialog into a new component?

**Review by kelsos — COMMENTED** [review]
Ok so first can we do something about the double scrollbar it doesn't look good?

image

I am using a 1920x1080 resolution btw.

**Comment by kelsos** [comment]
@lukicenturi So I have removed the auto-import changes and extracted the dialog 
What remains to be done to merge this is to fix the double scrollbar

**Comment by lukicenturi** [comment]
> @lukicenturi So I have removed the auto-import changes and extracted the dialog
> 
> What remains to be done to merge this is to fix the double scrollbar.
> 
> Make sure to pull my changes.

done

File: frontend/app/src/components/dashboard/EditBalancesSnapshotTable.vue [hunk 2]
```diff
@@ -20,6 +20,8 @@ import {
 import { One, Zero, bigNumberify, sortDesc } from '@/utils/bignumbers';
 import { isNft } from '@/utils/nft';
 import { toSentenceCase } from '@/utils/text';
+import { assert } from '@/utils/assertions';
+import ConfirmSnapshotConfictReplacementDialog from '@/components/snapshots/ConfirmSnapshotConfictReplacementDialog.vue';
```
Comment (kelsos) [inline frontend/app/src/components/dashboard/EditBalancesSnapshotTable.vue:24]: Doesn't the auto-import work in this case?

**Pull request merged**